### ISSUE TITLE
Upload folders with upload button (not drag and drop)

### ISSUE THREAD

**Issue by dbhagen** [owncloud/core#10997]
I've seen some bugs related to OS X bundles uploading, but this bug is not related to hard links or anything like that. The bundle uploads fine when the Desktop client syncs them, and if I drag the bundle into the browser window. They're treated as a folder, which is slightly annoying, but doable. But the upload button in the HTML interface causes errors: "The uploaded file was only partially uploaded"

My best guess would be that the upload button invokes the browser's file selector dialog but passes the selection as files and not folders. I've been out of web development for a while, so I'm not sure what HTML5 does for handling that in drag-and-drop vs. file picker dialog boxes.

Just hoping it's that the uploader button needs a parameter tweaked, or the upload function needs tweaked. Thanks!

**Comment by rgr101** [comment]
Yes, OS X bundles definitely are folders.

**Comment by PVince81** [comment]
This is a technical limitation, not a bug.

We could look into providing "webkit directory" upload feature http://sapphion.com/2011/11/html5-folder-upload-with-webkitdirectory/ but it is limited to Chrome.

Also I tried it locally and would need a separate button because it only lets you select folders (not files).

Marking as enhancement.

**Comment by RealRancor** [comment]
From https://github.com/owncloud/core/issues/16235

> Firefox expects to have it in the future:
https://bugzilla.mozilla.org/show_bug.cgi?id=846931
Perhaps we can vote importance to have it

**Comment by hodyroff** [comment]
As long as we support IE11 this will not be possible. Wake me up when that happens ;)

**Comment by trueuto** [comment]
> As long as we support IE11 this will not be possible. Wake me up when that happens ;)

Hi,

does it make sense to slow down the development because of one, already sunsetting browser? Especially it is now almost 2 years since the last post there and the functionality would be very useful.

Best regards,
TU

**Comment by hodyroff** [comment]
This is now working in Phoenix - the next generation web frontend for ownCloud - and will be working there. Closing here. Thank you so much for continuing here and reminding me of my previous statement.

Solution we found is that the button is not available for IE11 users. Everybody else has the button and will be happy.
IE11 is still not obsolete for many users worldwide and while I am fully with you, our task as enterprise file sync and share provider is to find the right compromises and proper paths through the market. I continue to look for your and others help and appreciate any feedback.
